Set user and group to havp:clamav by default in the config file

diff -Nur havp-0.92a.orig/etc/havp/havp.config.in havp-0.92a/etc/havp/havp.config.in
--- havp-0.92a.orig/etc/havp/havp.config.in	2010-04-02 08:33:34.000000000 -0500
+++ havp-0.92a/etc/havp/havp.config.in	2011-12-20 21:56:34.749035312 -0600
@@ -24,8 +24,8 @@
 # used by any other program.
 #
 # Default:
-# USER havp
-# GROUP havp
+USER havp
+GROUP clamav
 
 # If this is true HAVP is running as daemon in background.
 # For testing you may run HAVP at your text console.
